src/manifest_parser_perftest.cc

```diff
@@ -48,7 +48,7 @@ bool WriteFakeManifests(const string& dir, string* err) {
   if (mtime != 0)  // 0 means that the file doesn't exist yet.
     return mtime != -1;
 
-  string command = "python misc/write_fake_manifests.py " + dir;
+  string command = "./misc/write_fake_manifests.py " + dir;
```

Comment:
This should use the NINJA_PYTHON preprocessor define.

Reply:
Done, had to update the cmake to set it for these files as well